front/Dockerfile

```
@@ -1,8 +1,7 @@
-FROM node:22-bullseye-slim AS local
+ARG BUN_VERSION=1.1.22
+FROM oven/bun:${BUN_VERSION}-slim AS local
 WORKDIR /usr/src/twin-te/front
 
-RUN npm install -g bun@v1.1.22
-
 COPY package.json bun.lockb ./
 RUN bun install --frozen-lockfile
```

[nit] こっちのほうがpackage.jsonが変わったときにキャッシュが効きやすくなると思うけどどちらでも（以下同じ）

```suggestion
RUN --mount=type=cache,target=~/.bun/install/cache \
  --mount=type=bind,source=package.json,target=package.json \
  --mount=type=bind,source=bun.lockb,target=bun.lockb \
  bun install --frozen-lockfile
```

---

fixed https://github.com/twin-te/twin-te/pull/330/commits/95e0e621319daec929e658dd1bb047b844a961d1